blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineCreateRequestTest.java

```diff
@@ -368,6 +365,7 @@ public void shouldFindUserStoreCredential() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Note that #2503 may have helped, but the test was dubious to begin with. In fact CloudBees is experimenting with disabling Internet access during PCT runs, to reduce flakiness, which I think would cause this test to hang.